packages/babel-traverse/test/arrow-transform.js

```diff
@@ -518,6 +518,152 @@ describe("arrow function conversion", () => {
     );
   });
 
+  it("should convert super.prop operator logical assign `??=`", () => {
+    assertConversion(
+      `
+      () => {
+        super.foo ??= 4;
+      };
+      super.foo ??= 4;
+      () => super.foo ??= 4;
+    `,
+      `
+      var _superprop_setFoo = _value => super.foo = _value,
+          _superprop_getFoo = () => super.foo;
+
+      (function () {
+        _superprop_setFoo(_superprop_getFoo() ?? 4);
```

Comment:
Logical expressions short-circuit, so this should become something like
```js
_superprop_getFoo() ?? _superprop_setFoo(4);
```

Reply:
Ah, Thanks